demos/defender/aws_iot_demo_defender.c

```diff
@@ -1,6 +1,5 @@
 /*
- * Amazon FreeRTOS V201908.00
- * Copyright (C) 2019 Amazon.com, Inc. or its affiliates.  All Rights Reserved.
```

Comment:
We would need the License header with Amazon FreeRTOS V201908.00 info. This can be handled before release as well.

Reply:
OK